Sources/Error Handling/SKError+Extensions.swift

```diff
@@ -61,6 +61,21 @@ extension SKError {
             return ErrorUtils.storeProblemError(error: self)
 
         @unknown default:
+            switch self.code.rawValue {
+            case 907: // "Unhandled exception"
+                if let error = self.userInfo[NSUnderlyingErrorKey] as? NSError {
+                    switch (error.domain, error.code) {
+                    case ("AMSErrorDomain", 6): // "Payment Sheet Failed"
```

Comment:
same thought here regarding coding in the constants

Reply:
maybe we can add a comment saying something like "cancellations sometimes show up as undocumented errors instead of regular cancellations"